components/sendgrid/actions/delete-contacts/delete-contacts.js

```diff
@@ -0,0 +1,27 @@
+const sendgrid = require("../../sendgrid.app");
+
+module.exports = {
+  key: "sendgrid-delete-contacts",
+  name: "Delete Contacts",
+  description: "Allows you to delete one or more contacts.",
+  version: "0.0.7",
+  type: "action",
+  props: {
+    sendgrid,
+    deleteAllContacts: {
+      type: "string",
+      label: "Delete All Contacts?",
+      description: "Must be set to `true` to delete all contacts.",
+      optional: true,
+    },
+    ids: {
+      type: "string",
+      label: "Ids",
+      description: "A comma-separated list of contact IDs to delete.",
+      optional: true,
+    },
+  },
+  async run() {
+    return await this.sendgrid.deleteContacts(this.deleteAllContacts, this.ids);
```

Comment:
No, because specifiedly, both parameters are optional.
However, the user must provide one or the other.
`You must set either ids or delete_all_contacts.`
I changed the code force the `deleteAllContacts` prop to be boolean.